lib/dbutils.js

```
@@ -799,6 +799,26 @@ dbu.buildGetQuery = (req, options) => {
     return { cql, params };
 };
 
+/**
+ * CQL building for DELETE queries
+ * @param {InternalRequest} req
+ * @param  {object} options map
+ * @return {object} queryInfo object with cql and params attributes
+ */
+dbu.buildDeleteQuery = (req) => {
+    const schema = req.schema;
+    const query = req.query;
+    const attributes = query.attributes || {};
+    if (req.columnFamily !== 'meta') {
```

Hrmm, we do use this interface to (for example), write to `meta`.  See [lib/db.js#L657](https://github.com/wikimedia/restbase-mod-table-cassandra/blob/master/lib/db.js#L657), and the similar handling in [lib/dbutils.js#L656](https://github.com/wikimedia/restbase-mod-table-cassandra/blob/master/lib/dbutils.js#L656).  I can't think of any reason that we'd *need* to do deletes on `meta`, but I wasn't sure there was reason to exclude it either.

Do you think we should raise an exception instead?